doc/plugin_server_upstreamauthority_vault.md

```diff
@@ -0,0 +1,91 @@
+#  Upstream Authority "vault" Plugin
+
+The vault plugin requests to sign intermediate certificate (for signing X.509 SVIDs)to Vault PKI Engine.
+The plugin doesn't support `PublishJWTKeyRequest` and `PublishJWTKeyResponse`, 
+this means that you **SHOULD NOT** use `JWT SVID` in multiple SPIRE environment with this plugin.   
+
+## Configuration
+
+The plugin accepts the following configuration options:
+
+| key | type | required | description | default |
+|:----|:-----|:---------|:------------|:--------|
+| vault_addr  | string |   | A URL of Vault server. (e.g., https://vault.example.com:8443/) | `${VAULT_ADDR}` |
```

Comment:
> Is there ever a need for the URL to have anything other than the host and port, e.g., userinfo or a path component?

I've read in the documentation, it doesn't seem to be necessary .
Therefore I think we can set a host and port instead of the URL.

Reply:
I've tried to fix it, please check.
4394dd6